src/modules/Dropdown/DropdownMenu.js

```diff
@@ -8,13 +8,16 @@ import {
   META,
 } from '../../lib'
 
-function DropdownMenu(props) {
-  const { children, className } = props
-  const classes = cx('menu transition', className)
-  const rest = getUnhandledProps(DropdownMenu, props)
-  const ElementType = getElementType(DropdownMenu, props)
+export default class DropdownMenu extends Component {
+  render() {
+    const { children, className } = this.props
+    const classes = cx('menu transition', className, { visible: this.props.opened })
+    const rest = getUnhandledProps(DropdownMenu, this.props)
+    const ElementType = getElementType(DropdownMenu, this.props)
 
-  return <ElementType {...rest} className={classes}>{children}</ElementType>
+    return (<ElementType {...rest} className={classes}>
+      {children}</ElementType>)
```

Comment:
Not sure why the linter didn't catch this.  Please return all on one line without parens, or return like so:

```jsx
return (
  // <ElementType...
)